src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/WorkingDayCountServiceTest.kt

```diff
@@ -223,4 +224,19 @@ class WorkingDayCountServiceTest {
 
     assertThat(workingDayCountService.addWorkingDays(thursday, 10)).isEqualTo(expected)
   }
+
+  @Test
+  fun `getNextWorking day returns next working day`() {
+    every {
+      mockGovUKBankHolidaysApiClient.getUKBankHolidays()
+    } returns emptyBankHolidays
+
+    val date = LocalDate.now()
+
+    assertThat(
+      workingDayCountService.nextWorkingDay(date),
+    ).isEqualTo(
+      date.getNextWorkingDay(workingDayCountService.bankHolidays),
```

Comment:
Is it worth trying to be more literal here, e.g.

// Fri 1st March 2024
val date = LocalDate.parse(01-03-2024)

// Mon 4th March 2024
val expectedNextWorkingDate = LocalDate.parse(04-03-2024)

Reply:
I've fixed the tests to lean a bit less on mocks now, so hopefully they're more readable